static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx

```diff
@@ -112,8 +112,10 @@ const HoverList = styled('ul')`
   margin: 0;
   padding: 0;
 `;
-const HoverListItem = styled('li')`
-  display: flex;
+
+const HoverListItem = styled('ul')`
+  display: grid;
+  grid-template-columns: max-content 150px;
```

Comment:
Could we try 250px?

Reply:
It bleeds over 
<img width="361" alt="image" src="https://user-images.githubusercontent.com/7014871/174101313-b212fd8a-5b37-45d7-9d5d-b7164cf42ce2.png">

Here it is at 175
<img width="361" alt="image" src="https://user-images.githubusercontent.com/7014871/174101548-7bb7debe-76d3-474e-b35a-23070797212b.png">

Reply:
That seems like it could bleed over either way depending on the content.

Reply:
@Danondso can you try `225px`? (per [tooltip.tsx](https://github.com/getsentry/sentry/blob/2beeba84e9e9c8e8d4bb0b397ea5356d32868d25/static/app/components/tooltip.tsx#L392))

That 225px is less than the hovercard max of [295px](https://github.com/getsentry/sentry/blob/2beeba84e9e9c8e8d4bb0b397ea5356d32868d25/static/app/components/hovercard.tsx#L316), but the linebreaks are what we really need here.


@evanpurkhiser The change we need to make is to force linebreaks in the content, and have the popup be as wide as we can get. Is there another component combo that might this work?

Reply:
@ryan953 225px lookin' sad 
<img width="363" alt="image" src="https://user-images.githubusercontent.com/7014871/174314001-3a8bd6cb-9624-49aa-adbf-bbfb0b8f6a08.png">